### PR TITLE
chore(appactivity): Put AppActivityAPI behind feature flip

### PR DESCRIPTION
**Pull request by JustinHoldstock**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @JustinHoldstock has signed the CLA. Thanks for the pull request!

File: src/api/Feed.js [hunk 1]
```diff
@@ -175,7 +177,7 @@ class Feed extends Base {
         const versionsPromise = this.fetchVersions();
         const commentsPromise = this.fetchComments(permissions);
         const tasksPromise = shouldShowNewTasks ? this.fetchTasksNew() : this.fetchTasks();
-        const appActivityPromise = this.fetchAppActivity(permissions);
+        const appActivityPromise = shouldShowAppActivity ? this.fetchAppActivity(permissions) : Promise.resolve();
```
Comment (jeremypress) [inline src/api/Feed.js]: Is the empty state different from the 404 state?

Comment (JustinHoldstock) [reply src/api/Feed.js]: It should be the same, however the sort method prefers nothing compared to the Object literal

**Pull request merged**